portfolio.css

```diff
@@ -0,0 +1,75 @@
+* {
+	padding: 0;
+	margin: 0;
+	box-sizing: border-box;
+}
+
+body {
+	font-family: sans-serif;
+}
+
+#welcome-section {
+	background-color: #282c34;
+	min-height: 100vh;
+	display: flex;
+	align-items: center;
+	justify-content: center;
+	width: 100vw;
+}
+
+#navbar {
+	position: fixed;
+	top: 0;
+	right: 0;
+	padding: 0.8rem 1.5rem;
+	background-color: red;
+	width: 100%;
+	display: inline-flex;
+	justify-content: end;
+}
+
+#navbar a {
+	color: white;
+	text-decoration: none;
+	padding-right: 20px;
+}
+
+.header > h1 {
+ color: white;
+}
+
+#Contacts{
+    background-color:#303841;
+    min-height: 80vh;
+}
+#Contacts h1{
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    padding-top: 20vh;
+    color: white;
+    font-size: 60px;
+}
+p{
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    color: white;
+    font-size: 25px;
+}
+.icons{
+    display: flex;
+    justify-content:space-between;
+    align-items: center;
+    width: 40%;
+    margin-left: 30%;
+}
+.icons a{
+color: white;
+font-size: 20px;
+}
```

Comment:
It would be better if you give two spaces before the "{" tag and one empty line after "}".